pkg/kube_config_manager/backend/configmap/configmap.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"time"
 
-	log "github.com/sirupsen/logrus"
+	log "github.com/deckhouse/deckhouse/pkg/log"
```

Comment:
Our package and so-called log, do not need to set alice for it

Reply:
done